src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,130 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    static final Task[] TASKS = new Task[101];
+    static int tasksI = 1;
+
+    public static void main(String[] args) {
+        // print intro message
+        start();
+        // run bot (decode task)
+        run();
+    }
+
+    public static void start() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
```

Comment:
Good effort to extract such related behaviour into a seperate method. Perhaps consider converting the name into a verb form to better adhere to the convention (eg. printStartMessage)?